journal.py

```diff
@@ -108,6 +123,12 @@ def all(cls, session=None):
             session = DBSession
         return session.query(cls).order_by(cls.created.desc()).all()
 
+    @classmethod
+    def get_article(cls, article_id, session=None):
+        if session is None:
+            session = DBSession
+        return session.query(cls).filter(cls.id == article_id).all()
```

Comment:
This will return a list that contains, most likely, one article.  You could simplify things by using the `get` api on the query, or by using the `one` api.  Either would return a single entry *not* wrapped in a list.  And `one` would throw an error if there were more or less than one matching your filter.